mmv1/third_party/terraform/utils/node_config.go.erb

```diff
@@ -277,7 +277,6 @@ func schemaNodeConfig() *schema.Schema {
 				"tags": {
 					Type:     schema.TypeList,
 					Optional: true,
-					ForceNew: true,
```

Comment:
This file is also used in `google_container_cluster` resource. Do we know if the tags under `google_container_cluster` resource can also be updated. If not, we probably want to add it to `forceNewClusterNodeConfigFields` in `resource_container_cluster.go.erb` file at https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/resources/resource_container_cluster.go.erb#L78.

Reply:
Yes I see, Looking at the documentation 

`nodeConfig
(deprecated)`

Parameters used in creating the cluster's nodes. For requests, this field should only be used in lieu of a "nodePool" object, since this configuration (along with the "initialNodeCount") will be used to create a "NodePool" object with an auto-generated name. Do not use this and a nodePool at the same time. For responses, this field will be populated with the node configuration of the first node pool. (For configuration of each node pool, see nodePool.config)

It looks like the preferred method is to not define the `nodeConfig` within the `google_container_cluster`

Reply:
I believe if the tag can be updated by defining the `node_pool ` inside of the `google_container_cluster`, which means the in-place update functionality can be supported also in `google_container_cluster`, then we don't need to add this field to `forceNewClusterNodeConfigFields` (it should only contains the field that can be updated for `google_container_node_pool` but not `google_container_cluster`).

Reply:
Understood makes sense to me